bedrock/firefox/firefox_details.py

```diff
@@ -209,6 +209,10 @@ def get_download_url(self, channel, version, platform, locale,
         _locale = 'ja-JP-mac' if platform == 'osx' and locale == 'ja' else locale
         _platform = 'win' if platform == 'winsha1' else platform
 
+        # Bug 1340087 - Only include funnelcake params for Windows 32bit en-US builds by default.
+        include_funnelcake_param = (_platform in settings.FUNNELCAKE_PLATFORMS and
```

Comment:
have `funnelcake_id` be first. This will mean that the settings lists aren't checked until they need to be if a `funnelcake_id` exists.